webapp/src/views/logged-in-structure.vue

```diff
@@ -101,12 +101,7 @@ export default {
   },
   async created() {
     if (Auth.isAuthenticated()) {
-      if (!this.user) {
-        await this.getCurrentUser();
-      }
-      await this.getBeneficiaries();
-      await this.getTransactions();
-      await this.getMiddlemen();
+      await this.getCurrentUser();
     }
```

Comment:
Why is the data not being refreshed here? What if it's needed by other pages other than beneficiaries? @lydianish

Reply:
I had done that initially but then I figured if you want to refresh the data in the tables you have to refresh the browser. So instead, I put the refresh data in the individual pages. That way if you move among pages, it refreshes the data. I removed this so it won't be redundant but maybe I should put it back.

Reply:
Maybe it would be better to have a "refresh" button in the individual pages? So that it only refreshes the data it needs. i.e. if you're in the beneficiaries page, does it need to refresh the middlemen data?

But then again, if this is fixing an existing bug, then go ahead with it and maybe we can optimise it later.

Reply:
Actually, I used the middlemen data to display the name of the person who added the beneficiary. But yeah a refresh button sounds good. I propose to look into that later when we add other features to the tables like pagination and searches. Meanwhile, should I also put the refresh data back in the logged-in-structure?